gnome-extra/libgsf/libgsf-1.14.47.recipe

```diff
@@ -0,0 +1,128 @@
+SUMMARY="I/O abstraction for reading/writing compound files"
+DESCRIPTION="GSF was written as part of the Gnumeric project. It started in \
+2002 as a replacement for libole2 which Gnumeric was using to handle the \
+Microsoft Excel® file format, as libole2's code had become difficult to \
+maintain and was difficult to generalize. GSF was designed to be a more \
+general library for dealing with a number of different types of structured \
+data files and streams.
+
+With the release of Gnumeric 1.1.6 (July 2002), Gnumeric used GSF and libole2 \
+was orphaned."
+HOMEPAGE="https://github.com/GNOME/libgsf"
+COPYRIGHT="2002-2020 Jody Goldberg"
+LICENSE="GNU LGPL v2.1
+GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/GNOME/libgsf/archive/LIBGSF_1_14_47.tar.gz"
+SOURCE_FILENAME="libgsf-$portVersion.tar.gz"
+SOURCE_DIR="libgsf-LIBGSF_1_14_47"
+CHECKSUM_SHA256="ca958db8b91113804e1c8ec772adb7cb3e44f81d404cf941330d47f3e25ccbe7"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_64"
+
+libVersion="114.0.47"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+       libgsf$secondaryArchSuffix = $portVersion
+       lib:libgsf$secondaryArchSuffix = $libVersionCompat
+       lib:libgsf_1$secondaryArchSuffix = $libVersionCompat
+       "
+
+REQUIRES="
+       haiku$secondaryArchSuffix
+       lib:libgio_2.0$secondaryArchSuffix
+       lib:libglib_2.0$secondaryArchSuffix
+       lib:libgobject_2.0$secondaryArchSuffix
+       lib:libintl$secondaryArchSuffix
+       lib:libxml2$secondaryArchSuffix
+       lib:libz$secondaryArchSuffix
+       "
+
+PROVIDES_devel="
+       libgsf$secondaryArchSuffix = $portVersion
+       devel:libgsf_1$secondaryArchSuffix = $libVersionCompat
+       "
+
+REQUIRES_devel="
+	libgsf$secondaryArchSuffix == $portVersion base
+	"
+
+CONFLICTS_devel="
+	libgsf${secondaryArchSuffix}_devel
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
This shouldn't be needed, as gcc2 is broken we don't have the tools package like this on gcc2h